### DISCUSSION TITLE
IndexBar可以用在网页html页面吗？怎么用？微信nlanguage

### DISCUSSION THREAD

**Discussion by nissansz** [youzan/vant#11537] [Q&A]
### What problem does this feature solve?

IndexBar可以用在网页html页面吗？怎么用？微信nlanguage

### What does the proposed API look like?

IndexBar可以用在网页html页面吗？怎么用？微信nlanguage

**Comment by chenjiahan** [comment]
可以的，具体用法建议你参考 Vant 文档。如果对一些网页开发的基础知识不太熟悉，可以找一些资料或视频先学习下
